src/components/Verses/BookmarkedVersesList.tsx

```diff
@@ -69,18 +105,25 @@ const BookmarkedVersesList: React.FC = () => {
                 lang,
               )}`;
               return (
-                <Button
-                  href={getVerseNavigationUrlByVerseKey(verseKey)}
-                  className={styles.bookmarkItem}
-                  type={ButtonType.Success}
-                  shape={ButtonShape.Pill}
-                  key={verseKey}
-                  onClick={() => {
-                    logButtonClick('homepage_bookmark');
-                  }}
-                >
-                  {bookmarkText}
-                </Button>
+                <div key={verseKey} className={styles.bookmarkItem}>
+                  <Link href={getVerseNavigationUrlByVerseKey(verseKey)}>
+                    <a
+                      className={styles.linkButtonContainer}
+                      onClick={() => {
+                        logButtonClick('homepage_bookmark');
```

Comment:
nit: this component is shared between the home page and profile page so we need to rename the log key. Also we should extract the function outside